webapp/lib/view.dart

```diff
@@ -319,6 +358,10 @@ class ContentView {
       datasetLabels: ['restart']);
   }
 
+  String currentTabView() {
```

Comment:
I think it would make sense to turn this into a getter, something like:
```
String get currentTabView => _conversationTabLink.classes.contains('active') ? 'conversations' : 'systems';
```

Reply:
okay....sound good